packages/serverless-contracts/src/contracts/SQS/utils/messagesValidation.ts

```diff
@@ -0,0 +1,74 @@
+import {
+  getBodyValidator,
+  getMessageAttributesValidator,
+} from './getRecordsValidator';
+import { SQSContract } from '../sqsContract';
+import { SendMessageBuilderOptions, SqsMessage } from '../types';
+
+export const validateMessage = <Contract extends SQSContract>({
+  contract,
+  message,
+  options,
+  index,
+}: {
+  contract: Contract;
+  message: SqsMessage<Contract>;
+  options: SendMessageBuilderOptions<Contract>;
+  index?: number;
```

Comment:
Could this be required?

Reply:
No, the`validateMessage` is used in both `sendMessage` and `sendMessages`. In `sendMessage` there is only one message, so there is no `index`. 

`index` is used here only to print better error messages